support/yara_patterns/tools/pe/x86/installers.yara

```diff
@@ -127,6 +127,72 @@ rule install_creator {
 		$s01 at pe.overlay.offset
 }
 
+rule quick_batch_compiler_2x {
+    meta:
+		tool = "I"
+		name = "Quick Batch File Compiler"
+        version = "2.x"
+	strings:
+        $delphi_01 = "Runtime error     at 00000000"                                            // Common Delphi/Embarcadero
+  		$delphi_02 = "Access violation at address %p in module '%s'. %s of address %p" wide     // Found in almost all Quick Batch samples
+  		$s01 = "File is corrupt."
```

Comment:
Nitpick: Could you please re-format the two added rules? There are many mixtures of spaces and tabs in the indentation.